### PR TITLE
decouple UserAccounts from NSTemplateSets

### PR DESCRIPTION
**Pull request by xcoulon**
E2E Tests for https://github.com/codeready-toolchain/member-operator/pull/309
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -36,4 +35,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20211110143945-dec71f4e5073
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20211112102125-7cd9c79c4c42
```
Comment (MatousJobanek) [inline go.mod:40]: reminder

**Review by MatousJobanek — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (xcoulon) [reply go.mod:40]: yes ;)

**Pull request merged**